tests/app/questionnaire/test_rules.py

```diff
@@ -680,6 +680,104 @@ def test_id_when_rule_uses_passed_in_group_instance_if_present(self):  # pylint:
                 evaluate_when_rules(when, {}, answer_store, 3)  # passed in group instance ignored if present in when
                 patch_val.assert_called_with('Some Id', answer_store, 0)
 
+    def test_id_when_rule_answer_count_equal_0(self):
+        """Assert that an `answer_count` can be used in a when block and the
+            correct value is fetched. """
+        answer_group_id = 'repeated-answer'
+        when = [{
+            'answer_count': answer_group_id,
+            'condition': 'equals',
+            'value': 0,
+        }]
+
+        answer_store = AnswerStore({})
+
+        with patch('app.questionnaire.rules._answer_is_in_repeating_group', return_value=False):
+            self.assertTrue(evaluate_when_rules(when, {}, answer_store, 0))
+
+    def test_answer_count_when_rule_equal_1(self):
+        """Assert that an `answer_count` can be used in a when block and the
+            correct value is fetched. """
+        answer_group_id = 'repeated-answer'
+        when = [{
+            'answer_count': answer_group_id,
+            'condition': 'equals',
+            'value': 1,
+        }]
+
+        fname_answer = Answer(
```

Comment:
Did you want to refactor these like you did the other test?

Reply:
Sorry about that. Changed it and just re-running the tests before pushing